### DISCUSSION TITLE
Finding peak coordinates of enriched motif

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1656] [Questions / Documentation] [answered]
Hi Ryan,

Say I have a handful of motifs that were identified from `peakAnnoEnrichment` of differentially accessible peaks. Is there a way to pull out the specific peak coordinates (or sets of peak coordinates) of the enriched motif?

Thanks!

**Comment by rcorces** [comment, marked as answer]
does this answer your question?
https://github.com/GreenleafLab/ArchR/discussions/1644#discussioncomment-3743214
